ballerina/kafka_records.bal

```diff
@@ -214,19 +216,23 @@ public type ConsumerRecord record {|
 # + value - Anydata record content
 # + timestamp - Timestamp of the record, in milliseconds since epoch
 # + offset - Topic partition position in which the consumed record is stored
+# + headers - Map of headers included with the record
 public type AnydataConsumerRecord record {|
     anydata key?;
     anydata value;
     int timestamp;
     PartitionOffset offset;
+    map<byte[]|byte[][]|string|string[]> headers;
 |};
 
 # Subtype related to `kafka:AnydataConsumerRecord` record.
 #
 # + value - Record content in bytes
+# + headers - Headers as a byte[] or byte[][]
 public type BytesConsumerRecord record {|
     *AnydataConsumerRecord;
     byte[] value;
+    map<byte[]|byte[][]> headers;
```

Comment:
When the consumer creates the value for this if the headers are not available do we just create an empty map?

Reply:
Yes

Reply:
In that case shouldn't we just do something like this?

```
map<byte[]|byte[][]> headers = {};
```

Reply:
On second thoughts, this is fine.